library/iyzico-for-woocommerce-gateway-formobjectgenerate.php

```diff
@@ -107,21 +108,25 @@ public function generateBasketItems($items,$order) {
 			$product 	= wc_get_product($productId);
 			$realPrice  = $this->helper->realPrice($product->get_sale_price(),$product->get_price());
 
-			$basketItems[$keyNumber] = new stdClass();
+			if($realPrice && $realPrice != '0' && $realPrice != '0.0' && $realPrice != '0.00' && $realPrice != false) {
```

Comment:
php7 ve uzerinde `if ( $realPrice ) ...` yeterli olmali:

```
php > $x = 0;
php > echo $x == false;
1
php > echo $x == 0.0;
1
php > echo $x == '0';
1
php > echo $x == '0.0';
1
php >```

Reply:
@pvtmert PHP 5.4 - 7.x  arasında çalışması için bu şekilde bir geliştirme yapıyoruz. Ayrıca WooCommerce ve Wordpress  konfigürasyon bağlı olarak bazı durumlarıda gözden kaçırmamamız için gerekli koşullar yer alıyor. 
Teşekkürler.